File: packages/insomnia-httpsnippet/test/fixtures/output/c/libcurl/text-plain.c
CURL *hnd = curl_easy_init();

curl_easy_setopt(hnd, CURLOPT_CUSTOMREQUEST, "POST");
curl_easy_setopt(hnd, CURLOPT_URL, "http://mockbin.com/har");

struct curl_slist *headers = NULL;
headers = curl_slist_append(headers, "content-type: text/plain");
curl_easy_setopt(hnd, CURLOPT_HTTPHEADER, headers);

curl_easy_setopt(hnd, CURLOPT_POSTFIELDS, "Hello World");

CURLcode ret = curl_easy_perform(hnd);

File: packages/insomnia-httpsnippet/test/fixtures/output/javascript/fetch/text-plain.js
const fetchOptions = {
  mode: 'cors',
  method: 'POST',
  headers: {
    'content-type': 'text/plain',
  },
  data: 'Hello World',
};

fetch('http://mockbin.com/har', fetchOptions)
  .then(response => response.json())
  .then(data => console.log(data));
